totalopenstation/tests/test_xml.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from totalopenstation.formats import Feature, Point
+from totalopenstation.output.tops_xml import OutputFormat
+
+class TestXMLOutput(unittest.TestCase):
+
+    def setUp(self):
+        self.data = [
+            Feature(Point(12.8, 76.3, 56.2),
+                    desc='PT',
+                    point_name='TEST POINT',
+                    id=1),
+            Feature(Point(19.8, 26.3, 46.2),
+                    desc='PT',
```

Comment:
it would be good to add the "PO" and "ST" cases here.

Reply:
Done